java-checks/src/main/java/org/sonar/java/checks/serialization/SerializableFieldInSerializableClassCheck.java

```diff
@@ -157,7 +157,7 @@ private static boolean isTransientSerializableOrInjected(VariableTree member) {
       return true;
     }
     SymbolMetadata metadata = member.symbol().metadata();
-    return isAnnotatedWith(metadata, JAVAX_INJECT, "javax.ejb.EJB", "org.apache.wicket.spring.injection.annot.SpringBean");
+    return isAnnotatedWith(metadata, JAVAX_INJECT, "javax.ejb.EJB", "org.apache.wicket.spring.injection.annot.SpringBean", "javax.annotation.Resource", "jakarta.annotation.Resource");
```

Comment:
this line looks too long to me. You could extract it in an array.
```suggestion
    return isAnnotatedWith(EXCLUDED_ANNOTATIONS);
```